corehq/apps/smsbillables/migrations/0010_auto__chg_field_smsbillable_gateway_fee__chg_field_smsbillable_usage_f.py

```diff
@@ -0,0 +1,113 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+
+    def forwards(self, orm):
+
+        # Changing field 'SmsBillable.gateway_fee'
+        db.alter_column(u'smsbillables_smsbillable', 'gateway_fee_id', self.gf('django.db.models.fields.related.ForeignKey')(to=orm['smsbillables.SmsGatewayFee'], null=True, on_delete=models.PROTECT))
+
+        # Changing field 'SmsBillable.usage_fee'
+        db.alter_column(u'smsbillables_smsbillable', 'usage_fee_id', self.gf('django.db.models.fields.related.ForeignKey')(to=orm['smsbillables.SmsUsageFee'], null=True, on_delete=models.PROTECT))
+
+        # Changing field 'SmsGatewayFee.currency'
+        db.alter_column(u'smsbillables_smsgatewayfee', 'currency_id', self.gf('django.db.models.fields.related.ForeignKey')(to=orm['accounting.Currency'], on_delete=models.PROTECT))
+
+        # Changing field 'SmsGatewayFee.criteria'
+        db.alter_column(u'smsbillables_smsgatewayfee', 'criteria_id', self.gf('django.db.models.fields.related.ForeignKey')(to=orm['smsbillables.SmsGatewayFeeCriteria'], on_delete=models.PROTECT))
```

Comment:
i'm not sure how to handle all of these `on_delete=models.PROTECT`. this is something new in the south upgrade that we are going to see in a lot of places. i don't *think* it will change functionality, but we may want to double check